frontstage_api/resources/secure_messaging/get_message_list.py

```diff
@@ -17,6 +17,7 @@ class GetMessagesList(Resource):
     method_decorators = [get_jwt(request)]
 
     @staticmethod
+    @auth.login_required
```

Comment:
It looks like all API requests require basic auth. Is it worth creating two blueprints. One for the info endpoint that doesn't need any basic auth and the other blueprint which the `@auth.login_required` is done before every request?

It will make it less likely to miss the basic auth when adding new apis

Reply:
Potentially we should have different blueprint for the different sets of resources as well though not sure how useful that would be

Reply:
This change will be added as part of another pull request once we decide what pattern to implement. For the time being I have added tests to check each api requires authentication except /info

Reply:
Those tests are a good idea, the basic auth stuff isn't necessarily caught by the 100% coverage